main.tf

```diff
@@ -78,7 +74,7 @@ module "storage" {
   efs_access_point_path        = var.efs_access_point_path
   s3_force_destroy_on_deletion = var.s3_force_destroy_on_deletion
   vpc_id                       = local.vpc_id
-  subnets                      = local.private_subnets
+  subnets                      = [for s in local.private_subnets : s.subnet_id]
```

Comment:
this is a bit confusing as we are defining the subnets in 3 diff places, consider renaming to `private_subnets_ids`

Reply:
its just subnet_ids.... we dont care if its public or private.